packages/plugin-ext/src/api/model.ts

```diff
@@ -299,3 +299,42 @@ export interface WorkspaceEdit {
     edits: Array<ResourceTextEdit | ResourceFileEdit>;
     rejectReason?: string;
 }
+
+export enum SymbolKind {
```

Comment:
hello, does it conflict with https://github.com/theia-ide/theia/blob/b9d9bba4bf1ad1bccde86b02230f4b143be7d189/packages/plugin-ext/src/plugin/types-impl.ts#L1101 ? (like removing one or the other)

Reply:
@benoitf  I'm not sure whether it's a good idea.
If I understand correctly, the `model.ts` is an API for main<->ext communication. But the `types-impl.ts` contains the types for using by the plugins. Correct me if I'm wrong, please.

Reply:
@azatsarynnyy You are right, types `model.ts` is for main<->ext communication. `types-impl.ts` contains implementation described in `theia.d.ts`

Reply:
Are there any objections to merging this PR?